torchbenchmark/models/demucs/demucs/augment.py

```diff
@@ -77,7 +77,10 @@ def forward(self, wav):
         device = wav.device
 
         if self.training:
-            group_size = self.group_size or batch
+            if self.group_size is not None:
```

Comment:
Is this rewrite because we do not support Python's `or` semantics for `None`? If so, this seems a common pattern, and could be a usability improvement.

Reply:
`or`, TorchScript thinks this `or` makes `group_size` a boolean, even explicit annotation doesn't fix that. Thus resulting in a longer non-pythonic code.